docs/config.md

```diff
@@ -17,11 +17,19 @@ e.g. `user.name = "YOUR NAME"` is equivalent to:
 name = "YOUR NAME"
 ```
 
-Headings only need to be set once in the real config file but Jujutsu favors the
-dotted style in these instructions, if only because it's easier to write down in
-an unconfusing way. If you are confident with TOML then use whichever suits you
-in your config. If you mix the styles, put the dotted keys before the first
-heading.
+For a more complicated example,
+
+```toml
+[template-aliases]
+"format_short_id(id)" = "id.shortest(12)"
+```
+
+is equivalent to `template-aliases."format_short_id(id)" = "id.shortest(12)"`.
```

Comment:
If you like, you could make this even more complicated:

```
[colors]
"commit_id prefix" = { bold = true}
```
is equivalent to `colors."commit_id prefix".bold = true`

The point of that would be to also explain that the `{}` notation. Feel free to ignore this suggestion.

Reply:
I'm thinking of starting the release now, so let's get this PR merged and we can consider that for later

Reply:
Yes, it's a good idea, I'll turn it into a PR. Apparently, there's a bonus for landing the first merge after a release, so it might put me ahead of the curve :)

Reply:
Oh, so that's how contributions are ordered in the release notes, that matches indeed. How peculiar.